packages/boot/tools/liquidation.ts

```diff
@@ -130,21 +91,22 @@ export const makeLiquidationTestContext = async t => {
     const managerPath = `published.vaultFactory.managers.manager${managerIndex}`;
     const { advanceTimeBy, readLatest } = swingsetTestKit;
 
-    !priceFeedDrivers[collateralBrandKey] ||
-      Fail`setup for ${collateralBrandKey} already ran`;
-    priceFeedDrivers[collateralBrandKey] = await makePriceFeedDriver(
-      collateralBrandKey,
-      agoricNamesRemotes,
-      walletFactoryDriver,
-      // TODO read from the config file
-      [
-        'agoric1krunjcqfrf7la48zrvdfeeqtls5r00ep68mzkr',
-        'agoric19uscwxdac6cf6z7d5e26e0jm0lgwstc47cpll8',
-        'agoric144rrhh4m09mh7aaffhm6xy223ym76gve2x7y78',
-        'agoric19d6gnr9fyp6hev4tlrg87zjrzsd5gzr5qlfq2p',
-        'agoric1n4fcxsnkxe4gj6e24naec99hzmc4pjfdccy5nj',
-      ],
-    );
+    await 0;
```

Comment:
our convention is `await null;`. If you want to change it please bring up in #team-engineering.

Reply:
That's fine.  Any constant value seems as good as any other, but I understand the value of following a convention.

I confess to occasionally write `await 47` when I'm feeling flippant, because my spouse is a graduate of Pomona College and has explained to me at great length why 47 is the most ideal number. https://magazine.pomona.edu/pomoniana/2015/02/13/the-mystery-of-47/